### PR TITLE
✨ Add Name Parameter To Import Function

### PR DESCRIPTION
**Pull request by SteffenKn**
## What did you change?

The `createBpmnFromXml` function can now handle a name to give the process definition a custom name.

Related to:
- https://github.com/process-engine/process_engine/pull/81
- https://github.com/process-engine/bpmn-studio/pull/294
- https://github.com/process-engine/process_engine_contracts/pull/27

Needs to be published first:
- https://github.com/process-engine/process_engine_contracts/pull/27

## How can others test the changes?

- Please have a look at this PR: https://github.com/process-engine/process_engine/pull/81

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@essential-projects/core_contracts": "^1.0.0",
     "@essential-projects/http_node": "^1.0.0",
-    "@process-engine/process_engine_contracts": "^2.0.0",
+    "@process-engine/process_engine_contracts": "5.1.0-ed515abb-b2",
```
Comment (ElRaptorus) [inline package.json]: Hint:
You can just specific the name of the feature here, like so:
```
"@process-engine/process_engine_contracts": "feature~add_select_own_name"
```

This way, the contracts-project will always be installed with the most up-to-date version for the specified feature. 🙂

File: src/process_engine_controller.ts [hunk 2]
```diff
@@ -54,11 +54,11 @@ export class ProcessEngineController {
 
   public async createBpmnFromXml(request: any, response: any): Promise<void> {
     const context: ExecutionContext = request.context;
-
+    const name: string = request.body.name;
     const xml: string = request.body.xml;
```
Comment (LeoTT) [inline src/process_engine_controller.ts]:
```ts
const {name, xml} = request.body;
```

File: src/process_engine_controller.ts [hunk 3]
```diff
@@ -54,11 +54,11 @@ export class ProcessEngineController {
...
 
     try {
-      const result: any = await this.processEngineService.createBpmnFromXml(context, xml);
+      const result: any = await this.processEngineService.createBpmnFromXml(context, xml, name);
```
Comment (LeoTT) [inline src/process_engine_controller.ts]: 👍

**Comment by cmg-dev** [comment]
PR description does not match the things you changed.

**Comment by cmg-dev** [comment]
@ElRaptorus Please have a look at my comments.

I think you do not agree with some, but in general we should find those things in the future.

**Comment by SteffenKn** [comment]
Done.
Here you can find the issue: https://github.com/process-engine/process_engine_http/issues/12

**Pull request merged**